Readability.js

```diff
@@ -1058,7 +1067,6 @@ Readability.prototype = {
   _findNextPageLink: function(elem) {
     var uri = this._uri;
     var possiblePages = {};
-    var allLinks = elem.getElementsByTagName('a');
```

Comment:
Fixed, I've been a little too prone at cutting things it seems :)

I've added more tests to cover things I've refactored.